d_apply_handoff_coefficients.R

```diff
@@ -17,7 +22,7 @@ d_targets_list <- list(
                          full.names = TRUE) %>% 
         .[grepl("filtered", .)] %>% 
         .[grepl("DSWE1", .)] %>% 
-        .[grepl("v2023-08-17", .)]
+        .[grepl(Sys.getenv("collation_date"), .)]
```

Comment:
these built in date arguments are cool!

Reply:
Thank you! I'd love for us to chat more about best practices around these types of non-static variables. I'm using the .Renviron file in this case (which I don't track, because it holds the spam-ready email address of our ROSS account...), but could be a handy way to load globals without using a .yml.